### ISSUE TITLE
Unable to get status code: referred header information is missing

### ISSUE THREAD

**Issue by justinvoelker** [yiisoft/yii2-authclient#139]
### What steps will reproduce the problem?
Create a new project with yii2-authclient 2.1.1. A project that was upgraded from 2.0.6 to 2.1.1 continues to work correctly but a new project that pulls all new packages via composer does not.

### What's expected?
When clicking on the auth button, a window will popup asking for permission. The permissions are accepted, the authentication continues, the popup disappears and the process is complete.

### What do you get instead?
When clicking on the auth button, a window pops up asking for permission. The permissions are accepted, the window hangs for a second and finally returns an error: "exception 'yii\httpclient\Exception' with message 'Unable to get status code: referred header information is missing.' in /home/justinv3/public_html/example.com/dev/vendor/yiisoft/yii2-httpclient/Response.php:68"

### Additional info

| Q                | A
| ---------------- | ---
| Yii vesion       |2.0.9
| PHP version      |5.4.45
| Operating system |Linux

I can provide the complete stack trace upon request but as this issue is in regard the authentication I would rather not openly provide it here.

**Comment by klimov-paul** [comment]
Duplicates https://github.com/yiisoft/yii2-httpclient/issues/61

**Comment by justinvoelker** [comment]
Sorry for the duplicate I didn't see that and even if I did I would not have made the connection about them being related. Maybe I just need to wait until yii2-httpclient 2.0.2 is released, but, I manually made the couple changes specified in that issue to the StreamTransport.php file and now receive the following error: Argument 2 passed to yii\httpclient\Client::createResponse() must be of the type array, null given.

Perhaps that wasn't the fix for this?
